src/tests/ftest/util/job_manager_utils.py

```diff
@@ -251,10 +251,9 @@ def stop(self):
     def kill(self):
         """Forcibly terminate any job processes running on hosts."""
         regex = self.job.command_regex
-        result = stop_processes(self._hosts, regex)
-        if 0 in result and len(result) == 1:
-            self.log.info(
-                "No remote %s processes killed (none found), done.", regex)
+        result = stop_processes(self.log, self._hosts, regex)
+        if result.all_failed:
```

Comment:
`stop_processes` returns a `bool`, not result object

Reply:
I forgot to save the change, doh!